pandas/tests/frame/test_analytics.py

```diff
@@ -66,12 +66,15 @@ def assert_stat_op_calc(
     f = getattr(frame, opname)
 
     if check_dates:
+        expected_warning = FutureWarning if opname in ["mean", "median"] else None
```

Comment:
i think these moved?

Reply:
not sure what you mean.  from where?

Reply:
oh i see what you mean.  no, this one did not get moved yet.